### ISSUE TITLE
Be able to filter CSS/JS problems per file 

### ISSUE THREAD

**Issue by mlorant** [gmetais/YellowLabTools#109]
First of all: the concept is great :-)

It could be improved by adding a feature that allows to filter data per file. For example, in the "Different colors" section, I would like to see colors used in my project files and not those added by bootstrap and other plugins. 

In term of UI, there could be a two columns-list, with include & exclude files, like this: http://i.stack.imgur.com/icrtr.png

This would give a more accurate overview of the project status, since some plugins might be mandatory (= needed for a feature and don't want to reinvent the wheel) and include errors that you can't deal yourself (except by fixing them on the related project)

**Comment by gmetais** [comment]
Hi Maxime,

This is an excellent idea, that's definitely something that we should do!

Some rules could be improved quickly, as we already have the filename in the offenders.
Some other rules could be difficult to change and it might take some time.

**Comment by gmetais** [comment]
After digging into your proposal, I realized having a filtering system was too complex. It would need the filtering system to work on all of the rules. But some of the rules simply don't have the information about which file or files it is related to and it would be too much work to bring it up.

Take a look at #149. I changed the way some of the CSS offenders were displayed, and it's now much easier to visualize in which files the problems are.

I hope you'll find it convenient. Sorry but the "Different colors" you took as an example is not part of them :/
